cc-snapshot

```diff
@@ -90,8 +90,7 @@ then
                 	mv --backup=t "$SCRIPT" "$SCRIPT'-old'"
                 	mv "$NEWSCRIPT" "$SCRIPT"
                 	chmod +x "$SCRIPT"
-			exec "$SCRIPT" $1
-			exit $?
+			exec "$SCRIPT" $@
```

Comment:
The quoting is important to handle arguments containing spaces: "$@", not $@.

Reply:
Thank you! If all is good I will modify the url to the master branch, squash and merge.

Reply:
It looks good to me now. Can you modify the URL, squash (actually you may want to use fixup during the interactive rebase to avoid ending up with lots of duplicated lines in your commit messages), and force push so we can review the final commit?

Reply:
Yes! That looks better thanks.